src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/SubscriptionController.cs

```diff
@@ -0,0 +1,191 @@
+using System.Threading.Tasks;
+using Altinn.Common.AccessToken.Configuration;
+using Altinn.Common.PEP.Interfaces;
+using Altinn.Platform.Events.Authorization;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Services.Interfaces;
+using Altinn.Platorm.Events.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Controller to handle administration of event subscriptions
+    /// </summary>
+    [Route("events/api/v1/subscription")]
```

Comment:
Did you consider going with the plural form here? `subscriptions`

Reply:
Fixed